# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1362323274 18000
#      Sun Mar 03 10:07:54 2013 -0500
# Node ID d200b2b184c125b6120dbbfe388cf4dfa2cb7730
# Parent  0f0e970723ec31f579fae2cd44f15a5d5b7ee649
fix typo

* libinterp/Makefile.am (EXTRA_DIST): Fix typo.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -83,17 +83,17 @@ BUILT_NODISTFILES = \
   $(OPT_HANDLERS) \
   $(OPT_INC) \
   $(ALL_DEF_FILES) \
   builtins.cc
 
 EXTRA_DIST = \
   Makefile.in \
   DOCSTRINGS \
-  config-featurs.sh \
+  config-features.sh \
   find-defun-files.sh \
   gendoc.pl \
   genprops.awk \
   mk-errno-list \
   mk-pkg-add \
   mkbuiltins \
   mkdefs \
   mkops \
